packages/DropdownMenu/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paprika/dropdown-menu",
-  "version": "2.0.0",
+  "version": "0.1.5",
```

Comment:
https://www.npmjs.com/package/@paprika/confirmation there is not 0.1.5 version.  I think this should be 0.1.0 still and I will bump with lerna to 1.0.0 since we remove DropDownMenu.Confirmation

Reply:
😕 I've now added confirmation component as a dependency version 0.1.0. The version of the dropdown component i've left as 0.1.5 which is what it was pre https://github.com/acl-services/paprika/pull/160

Reply:
oohh yes my mistake I confused the components

Reply:
wait how did it go to 2.0.0?

Reply:
@alexzherdev I changed it in last PR by accident, Whoever is doing the paprika release will do the version bump